pkgs/applications/audio/blop-plugins/default.nix

```
@@ -0,0 +1,33 @@
+{ lib, stdenv, fetchurl, ladspa-sdk, ... }:
```

The `...` here isn't required:
```suggestion
{ lib, stdenv, fetchurl, ladspa-sdk }:
```